static/app/views/dashboardsV2/datasetConfig/base.tsx

```diff
@@ -43,7 +44,8 @@ export interface DatasetConfig<SeriesResponse, TableResponse> {
    */
   getTableFieldOptions: (
     organization: Organization,
-    tags?: TagCollection
+    tags?: TagCollection,
+    customMeasurements?: MeasurementCollection
```

Comment:
uhh I don't love the idea of adding customMeasurements to the common interface since its is only related to transactions data, but I don't have any suggestions around it since we can't retrieve them from the custom measurements store outside a component right? like we can get them inside `getEventsTableFieldOptions` in the ErrorsAndTransaction config right?

Reply:
Yeah it's not straightforward change to get the custom measurements in the config right now. I also made the change to use contexts as well so we'll need to somehow manage this within the provider. I think we can probably revisit this later once we get a little further into the custom measurements code? in order to not bloat this pr